static/src/javascripts/projects/commercial/modules/dfp/load-advert.ts

```diff
@@ -32,6 +32,7 @@ const eventTimer = EventTimer.get();
 
 export const loadAdvert = (advert: Advert): void => {
 	const adName = stripDfpAdPrefixFrom(advert.id);
+	// TODO can slotReady come after header bidding?
```

Comment:
If we are leaving this comment in could we add a bit more context pls around the implication of the answer?

Reply:
Great thanks!